python/cudf/dataframe/multiindex.py

```diff
@@ -250,6 +307,8 @@ def __next__(self):
             raise StopIteration
 
     def __getitem__(self, index):
+        if self._codes is None:
+            self._compute_levels_and_codes()
```

Comment:
Why do we need the codes calculated here?

Reply:
We don't!